src/index.js

```diff
@@ -21,6 +22,19 @@ winston.log('info', config);
 
 const app = express();
 
+let sessionOptions = {
+  secret: config.values.sessionSecret,
```

Comment:
For now, I just don't push my 'personal' config.

In the future, I'd like to be able to push the config encrypted in the repository or similar if possible. I.e. `git secret` or similar tools.